tutorials/EPL/synapses/GC->MC.yaml

```diff
@@ -0,0 +1,28 @@
+package: sapicore.engine.synapse.STDP
```

Comment:
Changed the default to double dash ("--"). Also added a list of valid synapse splitting strings to `utils.constants`. `split()` statements in `Network` have been updated to check against that list, ensuring that the component name only contains one valid splitter (erroring out otherwise). That way, if users have their own finicky preferences or constraints, they can simply edit the whitelist.